utils/scripts/ocm/ocm.py

```diff
@@ -95,7 +97,7 @@ def ocm_describe(self, filter=""):
             return None
         return ret
 
-    def is_osd_cluster_exists(self):
+    def is_rhoda_cluster_exists(self):
```

Comment:
I believe we can avoid rename as they are generic ones

Reply:
Reverted to generic naming